plutus-pab/src/Plutus/PAB/Arbitrary.hs

```diff
@@ -156,8 +157,53 @@ instance Arbitrary ThreadToken where
     shrink = genericShrink
 
 instance Arbitrary PlutusTx.Data where
-    arbitrary = genericArbitrary
-    shrink = genericShrink
```

Comment:
`genericShrink` is actually ok, it's from `QuickCheck` not `generic-arbitrary`.

Reply:
I added back the genericShrink, thanks for the heads up!